core-blocks/spacer/index.js

```diff
@@ -47,8 +47,8 @@ export const settings = {
 						} }
 						minHeight="20"
 						handleClasses={ {
-							top: 'core-blocks-spacer__resize-handler-top',
-							bottom: 'core-blocks-spacer__resize-handler-bottom',
+							top: 'block-spacer__resize-handler-top',
```

Comment:
I think you should update the stylesheet instead ;) These class names do not follow our guidelines ;)